content/2. Ondersteunende-informatie/4. Techniek/Git/3. Branches.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 3. Branches
+draft: false
+---
+
+> Meer informatie over branches: [Git branches](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell)
+## Wat zijn branches
+Een branch is een afzonderlijke lijn van ontwikkeling in een Git-repository. Bij het aanmaken van een repository komt altijd de branch `main` of `master` als hoofdbranch. Naast deze hoofdbranches kan je zelf ook branches erbij maken, zodat je kan werken aan de code zonder de hoofdcode te beïnvloeden.
+
+In een repository voor een codeer project is het gebruikelijk om in ieder geval een `main` en een `dev`/`development` branch te hebben. Bij een project kan dan elke nieuwe feature een eigen branch krijgen als voorbeeld.
+
+## Branches toevoegen
+Binnen een repository waar jij de rechten hebt om veranderingen te maken aan de repository, kan jij nieuwe branches aanmaken. Een nieuwe branch moet altijd gebaseerd worden op een andere branch (vaak de `main` of `development` branch).
```

Comment:
Ik weet niet of het klopt, dat moeten we even checken met @ernstbolt maar ik heb het gevoel dat `jij` niet wordt gebruik in de rest van het lesmateriaal. Moeten even letten op de woordkeuze dat het wel consistent gaat zijn

Reply:
Goed punt